vaccine_feed_ingest/runners/mn/gov/normalize.py

```diff
@@ -44,29 +45,34 @@ def _get_id(site: dict) -> str:
     return f"{runner}_{site_name}:{arcgis}_{layer}_{data_id}"
 
 
+def _cleanup_url(url):
+    if not url:
+        return None
+    # if "@" in url:
+    #     # Some of these are email addresses.
+    #     # Skipping those for now.
+    #     return None
+    if url == "COVID-19 Vaccine Information - Hennepin Healthcare":
+        return None
+    print(url)
+
+    url = re.sub(r"^https/:", "https://", url)  # fix typos
+    url = re.sub(r"^https:/t", "https://t", url)  # fix typos
```

Comment:
Why is the `t` in regex, that seems very specific. Perhaps changes this to be any non-slash character?

Reply:
the t is included because otherwise it was otherwise mangling legitimate urls. i.e. it would take a valid `https://` and make it `https:///`

Reply:
```suggestion
    url = re.sub(r"^(https?:\/)([^\/].*)", r"\g<1>/\g<2>", url)
```

I see, this regex is more generic and handles he cases:
1. Handle http or https
2. Escapes `/` as is the std
3. Matches any url missing `//` not just one that starts with `t`